x-pack/plugins/security/public/lib/effective_privileges/effective_privileges_utils.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import _ from 'lodash';
+
+// This is just a named set of actions.
+export interface ActionSet {
```

Comment:
nit: I don't think we need this here as the only use is by `PrivilegeScenario` which extends from `ActionSet`, can we just remove this and add these members to `PrivilegeScenario`?